scripts/send-to-bot.py

```diff
@@ -27,66 +26,60 @@
 
 ################################################################################
 
-def post_comment(pr_id, body, once):
-  endpoint = BOT_URL + '/comment'
-
-  if once:
-    endpoint += '/once'
 
-  headers = {'Content-Type': 'application/json'}
-  data = json.dumps({'pr_id': pr_id, 'body': body})
-  req = urllib.request.Request(endpoint, data.encode(), headers)
+def post_comment(pr_id, body, once):
+    endpoint = BOT_URL + "/comment"
 
-  try:
-    resp = urllib.request.urlopen(req)
-    code = resp.getcode()
-  except Exception as e:
-    print('Error sending data to tldr-bot:', str(e), file=sys.stderr)
-    return False
+    if once:
+        endpoint += "/once"
 
-  if code != 200:
-    print('Error: tldr-bot responded with code', code, file=sys.stderr)
-    print(resp.read(), file=sys.stderr)
-    return False
+    headers = {"Content-Type": "application/json"}
+    data = {"pr_id": pr_id, "body": body}
+    with requests.post(endpoint, data, headers) as r:
```

Comment:
```suggestion
    data = {"pr_id": pr_id, "body": body}
    with requests.post(endpoint, json=data) as r:
```

Using the `json=` named parameter will automatically handle setting the content-type as well as making sure the data is properly encoded.

Reply:
@MasterOdin I think we can apply the changes ourselves by now. Can this PR be merged after this suggestion?